x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx

```diff
@@ -26,20 +29,82 @@ import {
 } from '../../state_syncing/global_sync';
 import { AppStateManager } from '../../state_syncing/app_state_manager';
 import { startAppStateSyncing } from '../../state_syncing/app_sync';
-import { esFilters } from '../../../../../../../src/plugins/data/public';
+import {
+  esFilters,
+  Filter,
+  Query,
+  TimeRange,
+  IndexPattern,
+  SavedQuery,
+  QueryStateChange,
+  QueryState,
+} from '../../../../../../../src/plugins/data/public';
 import { MapContainer } from '../../../connected_components/map_container';
 import { getIndexPatternsFromIds } from '../../../index_pattern_util';
 import { getTopNavConfig } from './top_nav_config';
 import { getBreadcrumbs, unsavedChangesWarning } from './get_breadcrumbs';
+import {
+  LayerDescriptor,
+  MapRefreshConfig,
+  MapCenterAndZoom,
+  MapQuery,
+} from '../../../../common/descriptor_types';
+import { MapSettings } from '../../../reducers/map';
+
+interface MapsAppViewProps {
```

Comment:
Mind just naming `Props`? We have been using this convection when naming interfaces fro react properties to keep things simple and consistent across components.

Reply:
Fixed this.